Libraries/Components/View/View.js

```diff
@@ -28,13 +28,6 @@ const View: React.AbstractComponent<
   ViewProps,
   React.ElementRef<typeof ViewNativeComponent>,
 > = React.forwardRef((props: ViewProps, forwardedRef) => {
-  // [TODO(macOS GH#774)
-  invariant(
-    // $FlowFixMe Wanting to catch untyped usages
-    props.acceptsKeyboardFocus === undefined,
-    'Support for the "acceptsKeyboardFocus" property has been removed in favor of "focusable"',
-  );
-  // TODO(macOS GH#774)]
```

Comment:
Do we want to remove this now, or should we wait until the next big update?

Reply:
This will be part of 0.70, I think that's ok? It won't come in 0.68